src/services/member-service.ts

```diff
@@ -1,15 +1,40 @@
+// import { type } from "os";
 import logger from "../utils/logger/logger";
 
-// import SQLService from "./sql-service";
+import SQLService from "./sql-service";
 
 class MemberService {
 	static async isMember(student_id = "", email = "") {
 		logger.info(`student_id=${student_id} email=${email}`);
-
 		// Use SQLService to look up if student is a member.
 		// Either student_id or email may be empty. One is guaranteed to be valid.
 
-		return "Yes";
+		let SQLquery = "SELECT * FROM member ";
+		if (student_id != "") {
+			SQLquery += `WHERE uh_id=${student_id}`;
+		} else if (email != "") {
+			SQLquery += `WHERE email='${email}'`;
+		}
+		const result = await SQLService.query(SQLquery);
+		return result?.length == 0 ? "No" : "Yes";
```

Comment:
> I believe SQLService.query() always returns an array (that may be empty). Thus, you should be able to do result.length without the ?. However, definitely double check if I'm mistaken.
> 
> Also, I think you mean to use '===', since JS equals-to operators are somewhat odd.

Yeah I kinda just added the '?' in there just in case, but you're probably right. I looked in the SQLService class but honestly I'm confused about how the query function works. I'll change it though and look again. 

And thanks for noting the '===', I never understood when to use which but I just looked it up so I think I get it now...hopefully.

Reply:
> Does isMember() just check if a member was once a member? I believe it should check if it is a current member

Oh that's actually a good point. I didn't even consider this because I think all of the members I saw in the database were current members only, so I assumed that would always be the case. I'll check again though and add this feature if needed.